setup-py.sh

```diff
@@ -36,11 +36,7 @@ while test $# -gt 0; do
     shift
 done
 
-PYTHONEXECUTABLE=$PREFIX/bin/python
-
-pip install . \
-    --global-option build --global-option=--executable=$PYTHONEXECUTABLE \
-    --root $FAKEROOT                                                     \
-    --no-deps                                                            \
+pip install .           \
```

Comment:
Is it not so that it builds towards the Python version in the release instead of the one in the current environment?

Reply:
But then why are we not passing these arguments to pip install for packages fetched from pypi?

Reply:
my question exactly... wanted to make this pip installation more like the other one... i don't know

Reply:
Good question. Perhaps we should? For now we are building with a rather similar Python as we are targeting. So perhaps this is the reason it has worked out so far?

Reply:
Either way, I definitively support unification!